recipes/udatetime/meta.yaml

```diff
@@ -0,0 +1,78 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Would be nice to remove all the comments from the example recipe (see note above), so the file would be more readable.